book/src/external-attacher.md

```diff
@@ -10,6 +10,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
+[external-attacher v3.3.0](https://github.com/kubernetes-csi/external-attacher/releases/tag/v3.3.0) | [release-3.3](https://github.com/kubernetes-csi/external-attacher/tree/release-3.3) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/csi-attacher:v3.3.0 | v1.17 | - | v1.17
```

Comment:
Can you change recommended k8s version to 1.22?

Reply:
Done. @xing-yang